docs/cli.md

```diff
@@ -0,0 +1,103 @@
+# CLI Commands
+
+- `validate-batch`
+- `build`
+- `diff`
+- `csv2genocrate`
+- `validate-dataset`
+
+---
+
+## `validate-batch`
+
+```
+Usage: genocrate [OPTIONS] PATH
+
+  Validate a batch of files or bags for BagIt or MD5 compliance.
+
+Options:
+  -t, --validation-type [bagit|md5]
+                                  Type of validation to perform
+  -p, --parallel INTEGER          Number of processes to run in parallel
+  --skip-integrity-validation     Skip integrity validation and only validate
+                                  RO-Crate metadata
+  --data-folder TEXT              The folder where the data files are located.
+                                  Defaults to 'data'
```

Comment:
"where the data files are located within the batch"?? i.e is this just renaming the name of the folder as used by bagit? doesn't that break the bagit convention?